tests/unittest/_torch/misc/test_autotuner.py

```
@@ -291,6 +291,106 @@ def forward(self,
+            return gemm_fallback(*inputs)
+
+    x, w = torch.randn(M, 64), torch.randn(64, 128)
+    runners = [CrashedPreparationRunner()]
+    tuner = AutoTuner.get()
+    tuning_config = TuningConfig(dynamic_tensor_specs=(DynamicTensorSpec(
+        input_idx=0,
+        dim_idx=0,
+        gen_tuning_buckets=get_power_of_2_num_tokens_buckets,
+        map_to_tuning_buckets=next_positive_power_of_2), ), )
+    with autotune():
+        runner, tactic = tuner.choose_one(
```

Ruff (0.16.1)

[warning] Unpacked variable `runner` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)